nxapi/nxapi/nxparse.py

```diff
@@ -506,13 +507,13 @@ def commit(self):
         try:
             full_body = "\n".join(map(mapfunc,items)) + "\n"
         except:
-            print "Unexpected error:", sys.exc_info()[0]
-            print "Unable to json.dumps : "
+            print("Unexpected error:", sys.exc_info()[0])
+            print("Unable to json.dumps : ")
             pprint.pprint(items)
         bulk(self.es, items, index=self.cfg["elastic"]["index"], doc_type="events", raise_on_error=True)
         self.total_commits += count
         logging.debug("Written "+str(self.total_commits)+" events")
-        print "Written "+str(self.total_commits)+" events"
+        print("Written "+str(self.total_commits)+" events")
```

Comment:
since this mixes the code with py2 stuff and py3, i would suggest for the prints to use the `"".format()` method available in the string object.

This should then be changed to
```python
print("Written {} events".format(self.total_commits))
```